src/import_svg.cc

```diff
@@ -5,39 +5,65 @@
 #include "clipper-utils.h"
 #include "AST.h"
 
-Polygon2d *import_svg(const std::string &filename, const bool center, const Location &loc)
+Polygon2d *import_svg(const std::string &filename, const double dpi, const bool center, const Location &loc)
 {
 	try {
 		const libsvg::shapes_list_t *shapes = libsvg::libsvg_read_file(filename.c_str());
 
+		double scale = 1.0;
+
 		double x_min = 1.0/0.0;
 		double x_max = -1.0/0.0;
 		double y_min = 1.0/0.0;
 		double y_max = -1.0/0.0;
-		double width = 0;
 		double height = 0;
 		for (const auto& shape_ptr : *shapes) {
 			PRINTD("SVG shape");
 			const auto page = dynamic_cast<libsvg::svgpage *>(shape_ptr.get());
+			double unit_scale = 1.0;
 			if (page) {
-				width = page->get_width();
-				height = page->get_height();
+				const libsvg::length_t h = page->get_height();
+				switch (h.unit) {
+				case libsvg::unit_t::UNDEFINED:
+				case libsvg::unit_t::NONE:
```

Comment:
NONE doesn't look like it's used any longer

Reply:
It's used inside `parse_length()` -> `get_unit()` (in libsvg/util.cc) when parsing numbers that have no unit.